linux_os/guide/system/var_nonlocal_paths.var

```diff
@@ -0,0 +1,16 @@
+documentation_complete: true
+
+title: 'Explicit non-local paths'
```

Comment:
In general, I find `non-local paths` confusing.
`remote paths` seems clearer to me.

Reply:
@matejak What do you think about this? ^